decompy/RL/Action/PastAction.py

```diff
@@ -0,0 +1,20 @@
+from decompy.RL.Action import Action
+
+
+class PastAction(Action.Action):
+    """
+    Data Structure for an Action, particularly a list of past actions.
```

Comment:
It's not a list. It is a single action. And it isn't a past action. It is a PassAction. As in an llvm optimization pass